dice_ml/utils/helpers.py

```diff
@@ -20,7 +20,7 @@ def load_adult_income_dataset(only_train=True):
     :return adult_data: returns preprocessed adult income dataset.
     """
     raw_data = np.genfromtxt('https://archive.ics.uci.edu/ml/machine-learning-databases/adult/adult.data',
-                             delimiter=', ', dtype=str)
+                             delimiter=', ', dtype=str, invalid_raise=False)
```

Comment:
Why is the invalid_raise set to False here? It seems if set to True it correctly raises the exception.

Reply:
One of the adult dataset rows has an extra space or something like that, which makes the `genfromtext` function think that there is an extra column. This was raising an error. `invalid_raise` just skips that problematic row

Reply:
Got it. Thanks for clarifying.